scripts/popup/model/ConnectionMessages.js

```diff
@@ -59,6 +59,21 @@ sap.ui.define([], function () {
             });
         }
 
+        /**
+         * Send message through the given connection to find page element id's matching the given selector.
+         *
+         * @param {com.ui5.testing.model.Connection} oConnection the connection to use for sending messages
+         * @param {object} oItemSelector selector information
+         *
+         * @returns {Promise} a Promise resolving to the matche page elements
+         */
+        static checkElementCount(oConnection, oItemSelector) {
```

Comment:
I am not sure whether the chosen name is well-chosen. The name reflects the usage context and *not* the functionality the function provides. Thus, a better name would be something like `findElementIDs` -- analogously to the function `findElements`, which returns the full-blown elements.